packages/loaders/src/zarr/lib/utils.ts

```diff
@@ -81,7 +85,11 @@ export async function loadMultiscales(store: ZarrArray['store'], path = '') {
     const { datasets, axes } = rootAttrs.multiscales[0];
     paths = datasets.map(d => d.path);
     if (axes) {
-      labels = axes;
+      if (isAxis(axes)) {
+        labels = axes.map(axis => axis.name) as Labels<string[]>;
```

Comment:
Honestly was not sure what to do about this.  `Labels` is kind of `viv` specific and so it is used semi-internally as such, but then it is also a type used on the OMENGFF typescript stuff so this is a bit of a punt.

Reply:
What I'm getting at is that it is specific about what it expects because that is how things used to be with OMENGFF and also how we view things generally speaking internally.  But that is no longer the case.

Reply:
It's only specific with regard to the last two axes being `["y", "x"]`. This is a requirement of our tools still. We could not handle datasets which don't have the last to axes this way

Reply:
I think we could resolve this nicely with:

```typescript

function castLabels(dimnames: string[]) {
  // some assertion ?
  return dimnames as Labels<string[]>;
}

if (axes) {
  labels = castLabels(isAxis(axes) ? axes.map(ax => ax.name) : axes);
}
```

Reply:
Great suggestion.